python/instrumentation/openinference-instrumentation-bedrock/README.md

```diff
@@ -1,9 +1,73 @@
 # OpenInference AWS Bedrock Instrumentation
 
+Python autoinstrumentation library for AWS Bedrock calls made using `boto3`.
+
+This package implements OpenInference tracing for `invoke_model` calls made using a `boto3` `bedrock-runtime` client. These traces are fully OpenTelemetry compatible and can be sent to an OpenTelemetry collector for viewing, such as [Arize `phoenix`](https://github.com/Arize-ai/phoenix).
+
 [![pypi](https://badge.fury.io/py/openinference-instrumentation-bedrock.svg)](https://pypi.org/project/openinference-instrumentation-bedrock/)
 
 ## Installation
 
 ```shell
 pip install openinference-instrumentation-bedrock
 ```
+
+## Quickstart
+
+In a notebook environment (`jupyter`, `colab`, etc.) install `openinference-instrumentation-bedrock`, `arize-phoenix` and `boto3`.
+
+[You can test out this quickstart guide in Google Colab!](https://colab.research.google.com/github/Arize-ai/phoenix/blob/main/tutorials/integrations/bedrock_tracing_tutorial.ipynb)
+
+```shell
+pip install openinference-instrumentation-bedrock arize-phoenix boto3
+```
+
+Ensure that `boto3` is [configured with AWS credentials](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html).
+
+First, import dependencies required to autoinstrument AWS Bedrock and set up `phoenix` as an collector for OpenInference traces.
+
+```python
+from urllib.parse import urljoin
+
+import boto3
+import phoenix as px
+
+from openinference.instrumentation.bedrock import BedrockInstrumentor
+from opentelemetry import trace as trace_api
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk import trace as trace_sdk
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+```
+
+Next, we'll start a `phoenix` server and set it as a collector.
```

Comment:
Worth mentioning explicitly that phoenix does not send data over the internet. That concern/question has come up a few times in the support channel.

Reply:
@anticorrelator gonna merge. You can do as a follow-up